package.json

```diff
@@ -30,98 +31,85 @@
+    "run-all": "yarn workspaces foreach -vip --topological-dev --exclude @zwave-js/repo run",
+    "run-all:p": "yarn workspaces foreach -vip --exclude @zwave-js/repo run",
+    "run-zjs": "yarn workspaces foreach -vip --include zwave-js run",
+    "run-deps": "yarn workspaces foreach -vip --topological-dev --include \"@zwave-js/*\" --exclude @zwave-js/repo run",
+    "run-deps:p": "yarn workspaces foreach -vip --include \"@zwave-js/*\" --exclude @zwave-js/repo run",
+    "build": "yarn run-zjs build",
+    "build:full": "yarn run-deps build && yarn run-zjs build_full",
+    "clean": "yarn run-all clean",
+    "watch": "yarn run-all:p watch",
     "test:reset": "jest --clear-cache",
     "test:ts": "jest",
-    "test:ci": "yarn run test:ts -- --runInBand",
-    "test:watch": "yarn run test:ts -- --watch",
+    "test:ci": "yarn run test:ts --runInBand",
+    "test:watch": "yarn run test:ts --watch",
     "test": "yarn run test:watch",
-    "coverage": "yarn run test:ts -- --collect-coverage",
-    "coverage:ci": "yarn run test:ci -- --collect-coverage",
+    "coverage": "yarn run test:ts --collect-coverage",
+    "coverage:ci": "yarn run test:ci --collect-coverage",
     "show-coverage": "yarn run coverage && start ./coverage/index.html",
     "lint:ts": "eslint --ext .ts \"packages/*/src/**/*.ts\"",
     "lint:wotan": "wotan",
-    "lint:config": "lerna run lint:config",
+    "lint:config": "yarn run-all lint_config",
     "lint:configjson": "prettier -c \"packages/config/config/**/*.json\"",
-    "lint:zwave": "lerna run lint:zwave",
+    "lint:zwave": "yarn run-all lint_zwave",
     "lint": "yarn run lint:wotan && yarn run lint:ts && yarn run lint:configjson",
     "implemented_ccs": "yarn ts packages/maintenance/src/implementedCCs.ts",
-    "gh-cc-table": "yarn run implemented_ccs -- --flavor=github --only-incomplete",
+    "gh-cc-table": "yarn run implemented_ccs --flavor=github --only-incomplete",
     "toLogEntryOverview": "yarn ts packages/zwave-js/maintenance/checkCCToLogEntry.ts",
     "usbdebug": "cmd.exe /c \"test\\usbdebug.bat\"",
     "commit": "git-cz",
-    "release": "yarn run build && lerna version --no-commit-hooks",
-    "release:all": "yarn run build && lerna version --no-commit-hooks --force-publish",
-    "preversion": "release-script --lerna-check",
-    "version": "yarn install && release-script --lerna",
-    "postversion": "git push && git push --tags",
+    "release": "yarn run build && yarn release-script",
     "config": "yarn ts packages/config/maintenance/importConfig.ts",
     "docs": "docsify serve docs",
     "docs:generate": "yarn ts packages/maintenance/src/generateTypedDocs.ts",
     "test:perf": "yarn ts test/valuedb-perf.ts",
     "test:firmware": "yarn ts test/firmware-extraction.ts",
-    "ts": "node -r esbuild-register"
+    "//": "remove cross-env when https://github.com/evanw/esbuild/issues/1406 is fixed",
+    "ts": "cross-env ESBUILD_WORKER_THREADS=0 yarn node -r esbuild-register"
```

Comment:
Yarn ships with a portable shell so you don't actually need `cross-env` to do this anymore
https://github.com/yarnpkg/berry/tree/1bea22b3fab51b4c7757616708c511580c007947/packages/yarnpkg-shell

Reply:
Good to know, thanks!

Since you're here, there's one more blocker I need to solve before I can actually make the switch.

With `lerna` (fixed mode) it was trivial to bump all changed workspaces to a common version. I can't seem find an easy way to do this with the `version` plugin, except by controlling it with an external script:
https://github.com/yarnpkg/berry/discussions/3067